automake/post/rules/headers.am

```diff
@@ -43,3 +44,15 @@ install-headers: $(BUILT_SOURCES) install-headers-recursive
 
 install-headers-recursive:
 	$(nl-make-subdirs)
+
+define MaybePrintInstallHeadersHelp
+$(NL_V_AT)echo "  install-headers"
+$(NL_V_AT)echo "    Generate all configured public header artifacts for this"
+$(NL_V_AT)echo "    project and install them in DESTDIR on the build host"
+$(NL_V_AT)echo "    system."
+$(NL_V_AT)echo
+endef # MaybePrintInstallHeadersHelp
+
+define PrintInstallHeadersHelp
+$(call MaybePrintInstallHeadersHelp)
```

Comment:
no need to $(call ..) unless you need to bind or the function uses numbered args...